dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -536,7 +541,11 @@ private String getJSON(List<Contentlet> cons, HttpServletRequest request, HttpSe
 
 		for(Contentlet c : cons){
 			try {
-				jsonCons.put(contentletToJSON(c, request, response, render, user));
+			    JSONObject jo = contentletToJSON(c, request, response, render, user);
+		        if (c.getStructure().getStructureType() == Structure.STRUCTURE_TYPE_HTMLPAGE){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getStructure"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "STRUCTURE_TYPE_HTMLPAGE"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

Reply:
Same here:
```
if (BaseContentType.HTMLPAGE.equals(c.getContentType().baseType())) {
```